frontend/src/features/admin-form/create/logic/components/LogicContent/InactiveLogicBlock/FieldLogicBadge.tsx

```diff
@@ -32,7 +32,7 @@ export const FieldLogicBadge = ({ field }: FieldLogicBadgeProps) => {
           <Icon as={fieldMeta.icon} fontSize="1rem" />
         </Tooltip>
         {field.questionNumber ? <Text>{field.questionNumber}.</Text> : null}
-        <Text isTruncated>{field.title}</Text>
+        <Text noOfLines={1}>{field.title}</Text>
```

Comment:
is there any reason to prefer `noOfLines={1}` here rather than `noOfLines={0}`?

if we do not truncate, it looks like this:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/25571626/176834478-2638429a-7166-4486-bb20-2f6714b427b5.png">

whereas truncating will look like this:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/25571626/176834585-eae36ab3-e338-4447-a211-fd521e517e4a.png">

note also the delete button overlaps with the field title.